cmds/dummy-oauth/main.go

```diff
@@ -19,7 +19,7 @@ import (
 
 var (
 	address = flag.String("addr", ":8085", "address")
-	keyFile = flag.String("private_key_file", "build/test-certs/oauth.key", "oauth private key file")
+	keyFile = flag.String("private_key_file", "build/test-certs/auth2.key", "oauth private key file")
```

Comment:
What is the reason for changing the default key?

Reply:
Perhaps, it would makes sense to document the purpose of the files in build/test-certs/*

Reply:
As far as I know, `oauth.key` (and `oauth.crt`) are not used anywhere in the repo and should probably be deleted (especially since `oauth.crt` isn't actually the public key for `oauth.key`!).  The `auth2` key pair is what is used in all the local/dummy/development configurations, so it's almost always what a user wants to use.  I've added #682 to track separately from this PR.